### PR TITLE
spellcheck en.json files

### PR DESCRIPTION
**Pull request by Findus23**
As the en.json files are not checked that often, I quickly went through all files with my spellchecker.

Some of the changes are a bit opinionated, so it would be great if someone could quickly read over them.

:warning: As this changes some essential strings and therefore will invalidate the translations (or are they only invalidated on key changes?), this shouldn't be merged shortly before a release.

### CODE REVIEW

**Comment by sgiehl** [comment]
@Findus23 removed your last commit as it was apparently on the wrong branch

File: plugins/PrivacyManager/lang/en.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
         "DoNotTrack_Description": "Do Not Track is a technology and policy proposal that enables users to opt out of tracking by websites they do visit, including analytics services, advertising networks, and social platforms.",
         "DoNotTrack_Disable": "Disable Do Not Track support",
         "DoNotTrack_Disabled": "Matomo is currently tracking all visitors, even when they specified \"I do not want to be tracked\" in their web browsers.",
-        "DoNotTrack_DisabledMoreInfo": "We recommend to respect your visitors privacy and enable DoNotTrack support.",
+        "DoNotTrack_DisabledMoreInfo": "We recommend respecting your visitors privacy and enable DoNotTrack support.",
```
Comment (katebutler) [inline plugins/PrivacyManager/lang/en.json]: "visitors" needs a trailing apostrophe (to indicate possession) and the sentence seems quite awkward. How about:
`We recommend enabling DoNotTrack support to respect your visitors' privacy`

**Review by katebutler — COMMENTED** [review]
Hi @Findus23, most of this looks good, but I've suggested a couple of changes.

File: plugins/Installation/lang/en.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
         "JSTracking_Intro": "To track your web traffic with Matomo you need to make sure some extra code is added to each of your webpages.",
         "LargePiwikInstances": "Help for users with high traffic websites",
         "Legend": "Legend",
-        "LoadDataInfileRecommended": "If your Matomo server tracks high traffic websites (eg. > 100,000 pages per month), we recommend to try fix this problem.",
+        "LoadDataInfileRecommended": "If your Matomo server tracks high traffic websites (eg. > 100,000 pages per month), we recommend trying fix this problem.",
```
[the remark this answers is not in the document]
Comment (Findus23) [reply plugins/Installation/lang/en.json]: You are totally right!

File: plugins/Insights/lang/en.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
     "Insights": {
-        "PluginDescription": "Provides Insights about your traffic. Insights are available as dashboard widgets as well as a new icon in reports to let you see most important trends in your data.",
+        "PluginDescription": "Provides Insights about your traffic. Insights are available as dashboard widgets as well as a new icon in reports letting you see most important trends in your data.",
```
[the remark this answers is not in the document]
Comment (Findus23) [reply plugins/Insights/lang/en.json]: Sounds good. I changed it.

File: plugins/PrivacyManager/lang/en.json (hunk 1 above)
Comment (Findus23) [reply plugins/PrivacyManager/lang/en.json]: Sounds good. I think I saw more cases where possessive apostrophes were suggested, but I am quite insecure about placing them correctly (as they aren't used in German), so I didn't change them.

**Pull request merged**